cobalt/build/configs/android_common.gn

```diff
@@ -7,3 +7,5 @@ treat_warnings_as_errors = false
 # Overriding flag from //media/media_options.gni.
 # Cobalt doesn't use //third_party/ffmpeg.
 media_use_ffmpeg = false
+# Uncomment to update android_aar_prebuilt info file.
+# update_android_aar_prebuilts=true
```

Comment:
This feels more like it deserves its own documentation (probably on go/cobalt) and should not be included in the args.gn file

Reply:
Yeah, that is fine. I removed it.